src/m365/aad/commands/app/app-get.spec.ts

```diff
@@ -43,14 +45,16 @@ describe(commands.APP_GET, () => {
       }
     };
     loggerLogSpy = sinon.spy(logger, 'log');
+    sinon.stub(cli, 'getSettingWithDefaultValue').callsFake(((settingName, defaultValue) => defaultValue));
```

Comment:
It would have been better to have added this to the `before` hook. But let's leave for now.